BINDINGS.md

```diff
@@ -45,9 +45,7 @@ Some people ported raylib to other languages in form of bindings or wrappers to
 | [raylib-matte](https://github.com/jcorks/raylib-matte)                                   | 4.6-dev          | [Matte](https://github.com/jcorks/matte)                             | MIT                  |
 | [Raylib.nelua](https://github.com/AuzFox/Raylib.nelua)                                   | **5.0**          | [nelua](https://nelua.io)                                            | Zlib                 |
 | [raylib-bindings](https://github.com/vaiorabbit/raylib-bindings)                         | 4.5              | [Ruby](https://www.ruby-lang.org/en)                                 | Zlib                 |
-| [NimraylibNow!](https://github.com/greenfork/nimraylib_now)                              | 4.2              | [Nim](https://nim-lang.org)                                          | MIT                  |
```

Comment:
Because it was archived and announced EoL: https://forum.nim-lang.org/t/7368#66717 I didn't know about the unmaintained section, I will move them there.